tests/sentry/issues/test_occurrence_consumer.py

```diff
@@ -405,3 +406,14 @@ def test_culprit(self) -> None:
         message["culprit"] = "i did it"
         kwargs = _get_kwargs(message)
         assert kwargs["occurrence_data"]["culprit"] == "i did it"
+
+    def test_priority(self) -> None:
+        message = deepcopy(get_test_message(self.project.id))
+        kwargs = _get_kwargs(message)
+        assert kwargs["occurrence_data"]["initial_issue_priority"] == PriorityLevel.LOW
+
+    def test_priority_defaults_to_grouptype(self) -> None:
+        message = deepcopy(get_test_message(self.project.id))
+        message["initial_issue_priority"] = None
+        kwargs = _get_kwargs(message)
+        assert kwargs["occurrence_data"]["initial_issue_priority"] == PriorityLevel.LOW
```

Comment:
I think we need another test here where we pass a different initial priority to the default and validate that it is set as expected